### PR TITLE
ui:fixed the inconsistent footer style Fix #31

### PR DESCRIPTION
**Pull request by manthanmk66**
## Fixes Issue

 Closes #31 

## Changes proposed

Changed Footer Font to match with the website theme

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots

image

### CODE REVIEW

File: views/pages/index.ejs [hunk 1]
```diff
@@ -235,7 +235,7 @@
                     <div id = "scroll-value">&#x1f815</div>
                 </div>
 
-                <div class="text-sm items-center justify-center flex flex-col mb-24">
+                <div class="raleway text-sm items-center justify-center flex flex-col mb-24">
```
Comment (pooranjoyb) [inline views/pages/index.ejs:238]: this is literally a one word change, please make sure you fix these all too : 

- Reduce the `Scroll-to-top` button to smaller size as compared to desktop view.
- Fix the font in Chatbot, it should also be `raleway`

**Review by pooranjoyb — CHANGES_REQUESTED** [review]

File: views/pages/index.ejs (hunk 1 above)
Comment (manthanmk66) [reply views/pages/index.ejs:238]: Thank you for the feedback I will make the changes.

Comment (manthanmk66) [reply views/pages/index.ejs:238]: Done with the changes
image
image

**Review by pooranjoyb — APPROVED** [review]

**Comment by pooranjoyb** [comment]
Ready to merge @sristy17

**Pull request merged**